codecparsers/vp9quant.h

```diff
@@ -22,9 +22,9 @@ extern "C" {
 
 int clamp(int value, int low, int high);
 
-int16_t vp9_dc_quant(int qindex, int delta);
+int16_t vp9_dc_quant(int bit_depth, int qindex, int delta);
```

Comment:
Ok, I know your situation now. vp9quant.h did not refer any thing from vp9parser.
So it's ok to use this prototype. But we need clamp the value, make sure index is in right range.